code/datums/ai_law_sets.dm

```diff
@@ -11,22 +11,22 @@
 	..()
 
 /******************** Nanotrasen/Malf ********************/
-/datum/ai_laws/nanotrasen
+/datum/ai_laws/eris
 	name = "NT Default"
```

Comment:
Shouldn't this be renamed to "Eris Default"?

Reply:
GOOD POINT
will fix